tools/nightly-e2e-tests/monitors-gen/monitor.tf.src

```diff
@@ -0,0 +1,25 @@
+resource "datadog_monitor" ${{monitor_id}} {
+  name               = ${{monitor_name}}
+  type               = ${{monitor_type}}
+  tags               = ["service:com.datadog.ios.nightly", "env:instrumentation", "team:rumm", "source:ios", ]
+  message            = <<EOT
+@maciek.grzybowski@datadoghq.com
+@mert.buran@datadoghq.com
+@marius@datadoghq.com
+EOT
+  escalation_message = <<EOT
+<nil>
+EOT
+
+  query             = ${{monitor_query}}
+  notify_no_data    = false
```

Comment:
I guess this should be behind variable as well, because we want to notify if there is no data for 24h (with exemption of monitors when we want to check that we don't get particular value)

Reply:
Indeed, added a variable with default value set to `false`.